clustered/ehcache-clustered/src/assemble/docker/server/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM azul/zulu-openjdk-alpine:17-jre-latest
+
+RUN apk update && apk upgrade && apk add bash
```

Comment:
Do we need bash because of the start server script, or the entrypoint.sh?

Reply:
Yes to keep the content in sync as much as possible as what's done in enterprise... If a fix has to be made, easier like that.